index.html

```diff
@@ -8,7 +8,29 @@
     <title>Microverse</title>
 </head>
 <body>
-    <h1>Hello Microverse</h1>
+    <div class="nav-bar">
```

Comment:
[ ] Kyndly use a `nav` tag istead of a div, this will make your code more semantic.

Take  this as an exaple
![image](https://user-images.githubusercontent.com/95243996/191742093-40885146-32d3-4704-ad65-58070c50d2a3.png)